Content.Server/GameObjects/EntitySystems/DoorSystem.cs

```diff
@@ -1,10 +1,45 @@
 ﻿using Content.Server.GameObjects.Components.Doors;
+using JetBrains.Annotations;
 using Robust.Shared.GameObjects.Systems;
 
 namespace Content.Server.GameObjects.EntitySystems
 {
+    [UsedImplicitly]
     class DoorSystem : EntitySystem
     {
+        /// <summary>
+        /// Allows all players to open airlocks, despite not having access. External airlocks, so airlocks that are
+        /// open to space, are not included.
+        /// </summary>
```

Comment:
This summary seems like an incorrect copy-paste from AccessTypes.AllowAll.

```suggestion
        /// <summary>
        ///     Determines the base access behavior of all doors on the station.
        /// </summary>
```

Reply:
Oops, that was changed from a boolean into an enum but I forgot to change the summary.